recipes/librdata/all/conanfile.py

```diff
@@ -39,13 +39,6 @@ def validate(self):
         if is_msvc(self):
             raise ConanInvalidConfiguration(f"{self.ref} does not support Visual Studio. Please use MinGW or MSYS2.")
 
-    def build_requirements(self):
```

Comment:
I think this is necessary on macOS but `autotools.autoreconf()` in build() was failed CI.